src/components/Breadcrumb/Breadcrumb.scss

```diff
@@ -7,165 +7,128 @@
 // --------------------------------------------------
 // Breadcrumb styles
 
+$Breadcrumb-overflowButtonColor: $ms-color-themePrimary;
+$Breadcrumb-overflowButtonSize: 16px;
+$Breadcrumb-buttonHoverColor: $ms-color-themeDark;
+$Breadcrumb-itemMaxWidth: 160px;
+$Breadcrumb-itemMaxWidth-sm: 116px;
 
 .ms-Breadcrumb {
-  margin: 23px 0 1px 0;
+  @include ms-baseFont;
+  margin: 23px 0 1px;
 
-  &.is-overflow {
-    .ms-Breadcrumb-overflow {
-      display: inline;
-    }
+  .ms-Breadcrumb-list {
+    white-space: nowrap;
+    padding: 0;
+    margin: 0;
   }
-}
-
-.ms-Breadcrumb-chevron {
-  font-size: $ms-font-size-l;
-  color: $ms-color-neutralSecondary;
-  vertical-align: top;
-  margin: 10px 0;
-}
 
-.ms-Breadcrumb-list {
-  display: inline;
-  white-space: nowrap;
-  padding: 0;
-  margin: 0;
+  .ms-Breadcrumb-chevron {
```

Comment:
This is a bad idea. Please revert the nesting.  When you nest like this, you are unnecessarily scoping the rules:

instead of:
.ms-Breadcrumb-chevron

it translates into:
.ms-Breadcrumb .ms-Breadcrumb-chevron

Which is unnecessarily verbose, is more bytes over the wire, and provides no value since it is already specific enough.